surveys/sphinx-survey-2025.yml

```
@@ -203,37 +233,38 @@ questions:
   - "Stability"
   - "Variety of features"
   - "Other projects in my ecosystem were using it"
-  - "It was chosen by others on the project"
+  - "It was chosen by others or before I joined"
```

```suggestion
  - "It was chosen by others"
```
Obviously, if it was chosen before I joined, it was chosen by others. We can even leave out "on the project". This is basically to catch the case "I did not chose it, so cannot say why I chose it".

Alternatively, one could generalize, so that you get meaningful answers from people who were not involved in the decision but know the backgound.

> Why why was the the tool you use for your documentation chosen?
> ...
> - "I don't know"

With a fallback "I don't know"

---

I quite like the proposed `- "It was chosen by others or before I joined"`